external/wpa_supplicant/src/drivers/Make.defs

```diff
@@ -18,8 +18,11 @@
 
 CSRCS += \
 	drivers.c \
-	driver_common.c \
-	driver_t20.c
+	driver_common.c
+
+ifeq ($(CONFIG_SCSC_WLAN), y)
```

Comment:
The driver file will be built when CONFIG_SCSC_WLAN is enabled. But drivers.c has codes under CONFIG_DRVIER_T20 conditional. They make mismatch. I think we don't need to use CONFIG_DRIVER_T20.

Reply:
CONFIG_DRIVER_T20 is very much needed, if you look at the consistency in other CONFIG_DRIVER* checks across wpa_supplicant/src/drivers.c. If we drop this, then it is not known what will be used to check when declaring wpa_driver_t20_ops. An alternative is to use CONFIG_DRVIER_T20 in the Make.defs as well.

Reply:
How will you define it in Make.defs? Currently, you wrote it without conditional. But it should be under board-specific conditional. You can't define it always.

```
+5. Make sure you have defined *CONFIG_DRIVER_\<DRIVER_NAME\>* using *external/wpa_supplicant/wpa_supplicant/Make.defs*, as shown below:
 +```
 +CFLAGS += -DCONFIG_DRIVER_<DRIVER_NAME>
 +```
```

Last line should be as below.
```
ifeq ($(CONFIG_XXX), y)
CFLAGS += -DCONFIG_DRIVER_<DRIVER_NAME>
endif
```

Reply:
And as I said previous, you should resolve mismatched CONFIG.
Definition is under A but usages is under B. This is definitely wrong.